packages/components/src/tabs/src/Tabs.css

```diff
@@ -11,25 +11,24 @@
 /* TAB LIST */
 .o-ui-tab-list {
     display: flex;
-    gap: var(--o-ui-sp-2);
 }
 
 /* TAB LIST | ORIENTATION | HORIZONTAL */
 .o-ui-tabs-horizontal .o-ui-tab-list {
-    border-bottom: 2px solid var(--o-ui-b-alias-low-break);
+    border-bottom: 4px solid var(--o-ui-b-alias-low-break);
```

Comment:
On se crée tu une variable locale pour ça? c'est utilisé dans des calculs pis toute

Reply:
I'm not against it.